### PR TITLE
Version 19.0.0 → 20.0.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,107 @@ file.
...
+  For example, before this change, the following expression:
+
+  ```dhall
+  λ(x : Text) → Text/replace "a" "b" "a${x}"
+  ```
+
+  … would normalize to:
+
+  ```dhall
+  λ(x : Text) → Text/replace "a" "b" "b${x}"
```
Comment (basile-henry) [inline CHANGELOG.md:26]: I think before that change it normalized even further than that:

```suggestion
  λ(x : Text) → "b${x}"
```

Comment (Nadrieril) [reply CHANGELOG.md]: Wasn't it `λ(x : Text) → "b${Text/replace "a" "b" x}"` rather? It's still wrong but I would not expect us to have missed that

Comment (Nadrieril) [reply CHANGELOG.md]: Nah you're right we completely missed it

**Review by ehmry — APPROVED** [review]

**Review by basile-henry — APPROVED** [review]
:+1:

**Pull request merged**